src/pagination/pagination.ts

```diff
@@ -11,44 +20,45 @@ import {NgbPaginationConfig} from './pagination-config';
   host: {'role': 'navigation'},
   template: `
     <ul [class]="'pagination' + (size ? ' pagination-' + size : '')">
-      <li *ngIf="boundaryLinks" class="page-item"
+      <li *ngIf="boundaryLinks" class="page-item ngb-pagination-first"
         [class.disabled]="!hasPrevious() || disabled">
         <a aria-label="First" i18n-aria-label="@@ngb.pagination.first-aria" class="page-link" href
           (click)="!!selectPage(1)" [attr.tabindex]="(hasPrevious() ? null : '-1')">
           <span aria-hidden="true" i18n="@@ngb.pagination.first">&laquo;&laquo;</span>
         </a>
       </li>
 
-      <li *ngIf="directionLinks" class="page-item"
+      <li *ngIf="directionLinks" class="page-item ngb-pagination-previous"
         [class.disabled]="!hasPrevious() || disabled">
         <a aria-label="Previous" i18n-aria-label="@@ngb.pagination.previous-aria" class="page-link" href
           (click)="!!selectPage(page-1)" [attr.tabindex]="(hasPrevious() ? null : '-1')">
           <span aria-hidden="true" i18n="@@ngb.pagination.previous">&laquo;</span>
         </a>
       </li>
       <li *ngFor="let pageNumber of pages" class="page-item" [class.active]="pageNumber === page"
-        [class.disabled]="isEllipsis(pageNumber) || disabled">
+        [class.disabled]="isEllipsis(pageNumber) || disabled" [class.ngb-pagination-dots]="isEllipsis(pageNumber)">
```

Comment:
- should be `ngb-pagination-ellipsis` as it's the name of the symbol and in all APIs we use `ellipsis`
- there is only test case for when this class is present, but never checked that is is not present (should be same as for `.disabled`)